src/2025h2/const-generics.md

```diff
@@ -0,0 +1,42 @@
+# Const Generics
+
+| Metadata         |             |
+|:-----------------|-------------|
+| Point of contact | @BoxyUwU    |
+| Teams            | lang        |
+| Task owners      | @BoxyUwU    |
+| Status           | Proposed    |
+| Tracking issue   |             |
+| Zulip channel    | N/A         |
+
+## Summary
+
+Work towards stabilizing the remaining const generics functionality that was left out of the original `min_const_generics` feature.
+
+## Motivation & Status Quo
+
+The `min_const_generics` feature was stabilized with a number of limitations, while some have since been lifted there are still some things we do not support:
+- Inferred arguments to const parameters (`feature(generic_arg_infer)`)
+- Const parameters with arbitrary user-defined types (`feature(adt_const_params)`)
+- Non-concrete uses of associated constants in the type system (`feature(min_generic_const_args)`/`feature(associated_const_equality)`)
+
+When using const generics it is common to run into these limitations and be unable to move forwards, having to rewrite your code to use workarounds or not use const generics at all. This is a poor user experience and makes the language feel incomplete.
+
+### The next 6 months
+
+- Get `generic_arg_infer`'s reference PR pushed over the finish line
```

Comment:
Wearing project group hat: This is a change to the Rust reference? In that case, we need a team ask for the @rust-lang/spec team.

Reply:
Ok, maybe we remove this item :)

Reply:
no revert whammies?
```suggestion
```